### PR TITLE
Feature/add device info

### PR DESCRIPTION
**Pull request by Mee-gu**
add DeviceInfo to get device implementation limit

### CODE REVIEW

File: cocos/renderer/backend/DeviceInfo.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <string>
+
+#include "Macros.h"
+#include "base/CCRef.h"
+
+CC_BACKEND_BEGIN
+
+class DeviceInfo : public Ref
```
Comment (minggo) [inline cocos/renderer/backend/DeviceInfo.h]: DeviceInfo doesn't have to inherit from `Ref`.

**Comment by Mee-gu** [comment]
update review

**Comment by Mee-gu** [comment]
close it and reopen to enable CI test

**Comment by Mee-gu** [comment]
close it and reopen to enable CI test

**Pull request merged**